ecommerce/baskets/filters.py

```diff
@@ -0,0 +1,25 @@
+from django.utils.translation import gettext_lazy as _
+from django_filters import rest_framework as filters
+
+from baskets.models import Basket, BasketItem
+from customers.filters import CustomerFilter
+from products.filters import ProductFilter
+
+
+class BasketFilter(filters.FilterSet):
+    customer = CustomerFilter
```

Comment:
boyle bir kullanimi nereden ogrendik? kullanimi hakkinda bilgi verebilir misin?

Reply:
Hocam burda önceden bu filtrelemeyi oluştuduğum için import ederek kullandım birbiri ile ilişkili tablolar arsında, ve bir hata problem almadım basket ednpointi vs diğerlerinde geldiği için. Gözümden kaçmış hata almadığım için sanırım.